models.py

```diff
@@ -286,16 +293,87 @@ class GalleyProofing(models.Model):
                   'can give the proofreader to complete their task you can '
                   'add it here.',
     )
-    proofed_files = models.ManyToManyField('core.Galley')
-    notes = models.TextField()
-    annotated_files = models.ManyToManyField('core.File')
+    proofed_files = models.ManyToManyField('core.Galley', blank=True)
+    notes = models.TextField(blank=True)
+    annotated_files = models.ManyToManyField('core.File', blank=True)
+    
+    class Meta:
+        ordering = ('assigned', 'accepted', 'pk')
 
     def __str__(self):
         return 'Proofing for Article {0} by {1}'.format(
             self.round.article.title,
             self.proofreader.full_name(),
         )
 
+    def cancel(self):
+        self.cancelled = True
+        self.completed = timezone.now()
+        self.save()
+
+    def reset(self):
+        self.cancelled = False
+        self.completed = None
+        self.accepted = None
+        self.save()
+
+    def complete(self):
```

Comment:
+ `self.cancelled = False` ?
Unlikely to happen but still possible from a stale page

Reply:
Agreed.